### PR TITLE
[wallet] Remove addwitnessaddress

### PR DESCRIPTION
**Pull request by jnewbery**
Fully removes the `addwitnessaddress` RPC method, which was deprecated in V0.17

### CODE REVIEW

**Comment by jnewbery** [comment]
Travis was failing because of new flake8 warnings. I've added a new commit removing all flake8 warnings from touched tests.

**Review by promag — COMMENTED** [review]
Tested ACK 850908a, some comments though.

nit, commit ebec90a prefix could be `rpc` — doesn't touch wallet code?

File: doc/release-notes-14296.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+addwitnessaddress RPC method removed
+------------------------------------
+
+The `addwitnessaddress` RPC was added for segwit testing in version 0.13.1. It
```
[the remark this answers is not in the document]
Comment (jnewbery) [reply doc/release-notes-14296.md]: fixed

File: doc/release-notes-14296.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+was deprecated in version 0.17.0. This version fully removes the RPC method.
```
[the remark this answers is not in the document]
Comment (jnewbery) [reply doc/release-notes-14296.md]: fixed

**Comment by jnewbery** [comment]
Thanks for the corrections @promag . I've fixed them both.

**Comment by sipa** [comment]
utACK 2b91e42ece78c3f2ae4bf12fa71f461d05426a19

**Comment by promag** [comment]
Tested ACK 2b91e42, only change was fixing release notes.

**Pull request merged**